bin/check-cluster-snapshots.rb

```diff
@@ -20,6 +20,11 @@ def main
 
   role_arn = "arn:aws:iam::" + env("ACCOUNT_ID") + ":role/" + env("ROLE_NAME")
 
+  puts "#{Time.new} Will sleep for #{n} seconds"
+  sleep(rand(20))
+  puts "#{Time.new} Yielding now..."
+  yield
```

Comment:
I don't think you need this `yield` here - you only need that keyword if you're yielding control to a block that was passed in as a parameter. That's not the case in this context, so you can just delete the yield keyword (and the mention of "Yielding" in the `puts` above.